### DISCUSSION TITLE
link to R help after mouse over function

### DISCUSSION THREAD

**Discussion by fretwurst** [posit-dev/positron#4956] [Feedback] [answered]
In Positron, as in VS Code, I find it very convenient that when hovering over functions with the mouse, you get the function description in a small box. In VS Code, the function in this small box is additionally linked to entire help articles. Can I set this up similarly in Positron?

<img width="1969" alt="Bildschirmfoto 2024-10-08 um 23 08 45 (2)" src="https://github.com/user-attachments/assets/c1f1134e-b51d-4453-b547-c82f94f65b5a">

**Comment by juliasilge** [comment, marked as answer]
Thanks for this feedback! You can typically bring up the full Help view for your cursor position via <kbd>F1</kbd> but I don't believe we currently have anything to click in the hover help to get you there. Also, unfortunately <kbd>F1</kbd> for Help does not work in Quarto right now 😬 as outlined in https://github.com/posit-dev/positron/issues/4097.
